src/components/data/listing/Listing.js

```diff
@@ -705,6 +712,18 @@ function Listing(props) {
                 onClose={onRenameDlgClose}
                 onRenamed={onRenameDlgClose}
             />
+            <MoveDialog
+                path={path}
+                open={moveDlgOpen}
+                selectedResources={getSelectedResources()}
+                onClose={() => setMoveDlgOpen(false)}
+                onRemoveResource={(resource) => {
+                    const newSelected = selected?.filter(
+                        (sel) => sel !== resource?.id
+                    );
+                    setSelected(newSelected);
+                }}
```

Comment:
🤔 I might be missing something, but seems to me like we don't need to update the selections here in the listing if the user deselects something from the move dialog.

Reply:
🤔 Users select files / folders to move from the listing. So deselecting from listing made sense to me. Also I didn't see another way to update `selectedResources` prop for `MoveDialog`.

Reply:
Maybe in this case, it might make sense for the Dialog to manage its own `selectedResources` state, but it could be pre-populated from the listing's prop: `const [selectedResources, setSelectedResources] = useState(initialSelection);`

Reply:
Yeah, that seems to be what the Bag component is doing too.

Reply:
Items in a Bag are fetched from service and not from the listing. We can revisit this if there is specific feedback from user(s).